_source/logzio_collections/_log-sources/aws-control-tower.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Ship AWS Control Tower logs
+open-source:
+  - title: S3 Hook
+    github-repo: s3-hook
+logo:
+  logofile: aws-control-tower.png
+  orientation: vertical
+data-source: AWS Control Tower
+data-for-product-source: Logs
+templates: ["azure-deployment-event-hubs"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - aws
+order: 630
+---
+
+AWS Control Tower is a tool to control a top-level summary of policies applied to the AWS environment to ensure that the deployed resourced conform the policies of the AWS environment.
+
```

Comment:
Not sure about this description. Does this describe AWS Control Tower itself? Or the integration?